### ISSUE TITLE
EJS templating inside client/ directory

### ISSUE THREAD

**Issue by klaszlo** [DaftMonk/generator-angular-fullstack#795]
Hi,

Can I use ejs templating inside client directory?
By default in server/routes.js:24 file:
res.sendfile(app.get('appPath') + '/index.html');

The app.get('appPath') resolves to "client", so the res.sendfile gets "client/index.html" string.

If I change the sendfile function to render, it is not working anymore, 
because it searches inside server/views/ directory (so it is looking for server/views/client/index.html).

Basically I would like to know why res.sendfile and res.render looks for different path.
And where the magic happens?

Any pointer is much appreciated.

**Comment by leonardoanalista** [comment]
I needed the same and this is what I did:

bau: 
```
  app.set('views', config.root);
  app.engine('html', require('ejs').renderFile);
  app.set('view engine', 'html');
  app.use(compression());
```

then replace :
```
res.sendfile(app.get('appPath') + '/index.html');
```

with:
```
res.render(app.get('appPath') + '/index', {data: "this is my data to be processed inside index.html"});
```

**Comment by Awk34** [comment]
Closing due to inactivity.
